libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/utils/ModulesUtils.kt

```diff
@@ -24,6 +24,10 @@ internal fun paramsProvider(moduleParams: ModuleParams): Array<ModuleProviderArg
                 ThreadController::class.java,
                 moduleParams.threadController
             ),
+            ModuleProviderArgument(
+                Function0::class.java,
+                moduleParams.currentGeometryIndicesProvider
+            ),
```

Comment:
@RingerJK , @Guardiola31337, I'm struggling to understand if `defaultMapboxModuleCreator` is able to resolve a router with constructor which doesn't contain current geometry indices provider 🤔

Reply:
@LukasPaczos confirmed that I can add a parameter here because it's only used for internal Mapbox router, as described [in the docs](https://github.com/mapbox/mapbox-base-android/blob/master/annotations/src/main/java/com/mapbox/annotation/module/MapboxModule.kt#L29).